app.py

```diff
@@ -1,12 +1,35 @@
 import os
-from flask import Flask
-from peewee import SqliteDatabase
-
-
-
+from flask import Flask,session,request
+import sqlite3
+import json
+from pyhunter import PyHunter
 app = Flask(__name__)
+from views import *
```

Comment:
`import *` is bad practice because it's hard to tell where names come from and it's really easy to have name clashes where the imported file is big. We should always do proper named imports.